pkg/daemons/control/server.go

```diff
@@ -749,3 +765,22 @@ func setupStorageBackend(argsMap map[string]string, cfg *config.Control) {
 		argsMap["etcd-keyfile"] = cfg.Storage.KeyFile
 	}
 }
+
+func expired(certFile string) bool {
+	certBytes, err := ioutil.ReadFile(certFile)
+	if err != nil {
+		return false
+	}
+	certificates, err := certutil.ParseCertsPEM(certBytes)
+	if err != nil {
+		return false
+	}
+	expirationDate := certificates[0].NotAfter
+	diffDays := expirationDate.Sub(time.Now()).Hours() / 24.0
+	fmt.Println(diffDays)
+	if diffDays <= 30 {
```

Comment:
I'd probably do 90 days

Reply:
Does this only check on service startup? If so 90 days seems way safer than 30. I know I go way more than a month without restarting my nodes.